docs/HACKING.md

```diff
@@ -7,6 +7,8 @@
 - The operator is written using operator-sdk 
 [v1.6.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.6.1) and has the same 
 [pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it does.
+- Login to  `podman login quay.io` with encrypted password from quay account.
+-  To generate encrypted password, login to your user account at quay.io and click `Generate Encrypted Password` from the account setting
```

Comment:
I wonder if this has a place here. This is general knowledge that needs to be known when it comes to dealing with container registries. This seems like it would belong more in an internal onboarding doc than within this repo.

Its also very specific to quay, when the user could use any other container registry.

Reply:
because this is kind of living document and we prefer to use Encrypted password  I wanted to mention it here. not sure how are we enforcing it if we use other registry.

Reply:
I agree with @sebsoto. The onboarding doc might be a better spot for this. We shouldn't be documenting how to use a registry here.

Reply:
I updated on-boarding document with above info and removed it from this document.